src/ripple/net/impl/RPCCall.cpp

```diff
@@ -1219,9 +1217,24 @@ int fromCommandLine (
         nRet                    = rpcINTERNAL;
     }
 
-    std::cout << jvOutput.toStyledString ();
+    return { nRet, std::move(jvOutput) };
+}
+
+//------------------------------------------------------------------------------
+
+namespace RPCCall {
+
+int fromCommandLine (
+    Config const& config,
+    const std::vector<std::string>& vCmd,
+    Logs& logs)
+{
+    auto const result = rpcClient(vCmd, config, logs);
+
+    if (result.first != 1)
```

Comment:
Actually, `1` means "an error occurred` and becomes the return value of `main`. We could use `EXIT_FAILURE` and `EXIT_SUCCESS` here (but that would require auditing the call chain of `rpcClient`).